src/lib/notify.js

```diff
@@ -140,10 +141,6 @@ class Notification extends BaseElement {
   }
 
   openSurvey(surveyUrl, sentiment, sitename) {
-    if (!storage.id) {
-      storage.id = {};
-    }
-    storage.id[this.id] = tabs.activeTab;
```

Comment:
I'm sure this is fine, but it just makes me wonder why this was necessary before but not any more. Is this supposed to be calling storeTab here too?